package.json

```diff
@@ -14,36 +14,44 @@
     "publishManually": "scripts/publishManually",
     "switchPostInstall": "scripts/switchPostInstall",
     "prettier:fix": "npx prettier --write --ignore-unknown .",
-    "prettier": "npx prettier --check ."
+    "prettier": "npx prettier --check .",
+    "test": "scripts/test"
   },
   "author": "Relaying Services Team",
   "license": "MIT",
   "dependencies": {
     "@rsksmart/rif-relay-client": "https://github.com/infuy/rif-relay-client",
     "@rsksmart/rif-relay-common": "https://github.com/infuy/rif-relay-common",
     "@rsksmart/rif-relay-contracts": "https://github.com/anarancio/rif-relay-contracts",
+    "@types/secp256k1": "4.0.2",
     "@types/node": "^13.0.0",
     "@types/semver": "^7.3.4",
     "@types/web3": "1.2.2",
     "@types/web3-provider-engine": "^14.0.0",
+    "bn.js": "5.1.2",
     "web3": "1.2.6",
     "web3-core": "1.2.6",
     "web3-core-helpers": "1.2.6",
     "web3-eth": "1.2.6",
     "web3-eth-abi": "1.3.0",
     "web3-eth-contract": "1.2.6",
-    "web3-utils": "1.2.6"
+    "web3-utils": "1.2.6",
+    "secp256k1": "^4.0.2",
+    "tslib": "^2.3.1"
   },
   "devDependencies": {
+    "@types/jest": "^26.0.15",
     "@typescript-eslint/eslint-plugin": "^4.28.3",
     "@typescript-eslint/parser": "^4.28.3",
     "eslint": "^7.30.0",
     "husky": "^6.0.0",
+    "jest": "^26.6.1",
     "pinst": "^2.1.6",
     "prettier": "2.3.0",
+    "ts-jest": "^26.4.2",
     "ts-loader": "^9.2.3",
     "ts-node": "8.6.2",
-    "typescript": "^4.3.5",
+    "typescript": "^4.2.3",
```

Comment:
For the sake of curiosity: why did we downgrade typescript?

Reply:
This downgrade was upload by mistake, it will revert.